app/controllers/floating_ip_controller.rb

```diff
@@ -14,4 +14,260 @@ def self.display_methods
   end
 
   menu_section :net
+
+  def button
```

Comment:
@martinpovolny,

All the network UI and others (cloud_tenant, cloud_volumes) use the exact same approach.
So I'm keen to make a refactor on all of them using a consistent approach.

That said, I don't see the point of a case for replacing a if/elsif. 
No idea about the @refresh_div, I haven't digged into it, what's for? What's the issue here?
Thanks